source/class/cv/io/mqtt/Client.js

```diff
@@ -129,9 +130,17 @@ qx.Class.define('cv.io.mqtt.Client', {
      *
      */
     subscribe(addresses, filters) {
+      this.addresses = addresses ? addresses : [];
```

Comment:
Wouldn't be the more modern `addresses ?? []` be better?

Reply:
Better? No, its just syntax and I'm old school ;-)